ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
```

Comment:
the go version for 4.17(main) is 1.22

Reply:
The configuration script asks for the go version of the repo. It automatically adds it here. It should be backwards compatible, so the build should work. The pf-rehearse passed.

Reply:
there is the all work done in openshift 4.17 to build with go toolchain 1.22.4 that is why I think for main (that points to 4.17) to have 1.22 if possible

Reply:
alright, will change it

Reply:
@SchSeba, I set it back to 1.21 - CI was complaining that golang-1.22 does not exist. This is the initial configuration. I will use `rhel-9-golang-1.22-openshift-4.17` in the following pr when I set up publishing images to quay.